City-of-Pori/pori-kada
# PORI-184: Attraction card slideshow & Fix news keywords field required

Attraction card slideshow & Fix news keywords field required.

Slideshow done with slick-slider since the flexslider caused problems with scald atoms.

To test:
1. drush updb -y && drush fra -y && drush cc all
2. create attraction card with multiple images
3. confirm that there is a slideshow on the attraction card page

Note: In addition there is a change to set keywords as a required field in news content-type.
